README.md

```diff
@@ -74,30 +74,123 @@ An example of one such record:
 }
 ```
 
+
 ## Making Queries
 
+
 All queries to the index are made through HTTP using JSON data payloads.
 This gives a simple means of interaction that is easily accessible to any
 number of languages.
 
 These queries are handled via requests and wrapped into the index client.
 
-### Create a record
 
-***TODO***
+  ### Create a record 
+
+
+### Method: create
```

Comment:
I would make this `####` so it's under "create a record" (same with other methods below)